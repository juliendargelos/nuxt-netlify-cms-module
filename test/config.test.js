/* eslint-disable import/imports-first */
jest.mock("../lib/utils/yaml");

import { get, commonBefore, commonAfter } from "./nuxt";

describe("config", () => {
  beforeAll(commonBefore());

  afterAll(commonAfter);

  test("netlify-cms.yml", async () => {
    const config = await get("/admin/config.yml");
    expect(config).toMatchSnapshot();
  });
});
