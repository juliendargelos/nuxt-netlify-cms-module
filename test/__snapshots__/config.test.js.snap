// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config netlify-cms.yml 1`] = `
"backend:
  name: github
  repo: me/test
  branch: master
media_folder: test/fixture/static/uploads
collections:
  - name: pages
    label: Page
    folder: test/fixture/content/pages
    slug: '{{slug}}'
    create: true
    fields:
      - label: Title
        name: title
        widget: string
        tagname: h1
      - label: Body
        name: body
        widget: markdown
    meta:
      - label: SEO Description
        name: description
        widget: text
"
`;
